### PR TITLE
Country Update

### PR DESCRIPTION
**Pull request by nikh050**
Added and edited comms lists for Iceland, Indonesia, and Honduras

### CODE REVIEW

File: resources/asia/indonesia/id-telegram-osm.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "IDosm",
+  "type": "telegram",
+  "locationSet": {"include": ["id"]},
+  "languageCodes": ["en"],
+  "name": "OpenStreetMap Indonesia",
+  "description": "OpenStreetMap Indonesia telegram chat",
+  "url": "https://t.me/joinchat/IJvRUw_VWY4tz-ZkRMSVcg",
```
Comment (kmpoppe) [inline resources/asia/indonesia/id-telegram-osm.json:8]:
```suggestion
  "url": "https://https://t.me/osmindonesia",
```

Comment (kmpoppe) [reply resources/asia/indonesia/id-telegram-osm.json:8]: If there's a short-URL it might make sense to use it, right?

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See inline comment

**Review by kmpoppe — APPROVED** [review]

**Comment by kmpoppe** [comment]
@bhousel I'd like your opinion on these, some entries have very recent activity (2020), other 2016/2017. Your take (#423) was that 404's should be removed (obvs!) and very inactive ones (do we need a policy as to what "very inactive" is) shouldn't be added?

I'm ambivalent here. I wouldn't want to add "old" stuff, but on the other hand "the more the merrier", right?

Kai

**Comment by bhousel** [comment]
> very inactive ones (do we need a policy as to what "very inactive" is) shouldn't be added?

We should be pretty understanding about the current situation with COVID - I would expect that a lot of meetups can't meet, and most YouthMappers chapters probably aren't doing anything, especially where universities are remote.  So even if a group hasn't done anything in the past year or two, it's reasonable to leave them in the index.

I think if a group hasn't done anything in 3 years it can be considered dormant and removed.  

Another point - I see people are still adding contact infos like "Talk-is-owner@"..  My intention with the `contact` field was to require a way to get in touch with an actual human responsible for the community, but contributors just added these generic emails to get around the requiredness of it, so I made it optional in 1ae6f8611a.  I think we should remove generic contacts that don't lead to a real person.

**Comment by kmpoppe** [comment]
> I think if a group hasn't done anything in 3 years it can be considered dormant and removed.

We might need a way to keep track of activity somehow to know what to do - I wouldn't want to find anyone who checks all the communitys manually.

> Another point - I see people are still adding contact infos like "Talk-is-owner@".. My intention with the `contact` field was to require a way to get in touch with an actual human responsible for the community, but contributors just added these generic emails to get around the requiredness of it, so I made it optional in [1ae6f86](https://github.com/osmlab/osm-community-index/commit/1ae6f8611a86e17b0fd06c6b15b32150466df3ef). I think we should remove generic contacts that don't lead to a real person.

Again, we'd need a list of owners to look at - because then we could manually add a real person bit by bit and only remove those that don't come back with a fitting answer to a question :) ... Much footwork again :-/

**Comment by bhousel** [comment]
btw, sorry my comment wasn't clear but the PR is fine.  I wasn't trying to hold it up.. 😭

**Comment by kmpoppe** [comment]
Nvm @bhousel, we should've moved that discussion to somewhere else :-)
Thanks for everything 😊!

**Pull request merged**